ui/src/app/components/resources.tsx

```diff
@@ -9,12 +9,13 @@ import {ResourceType} from 'app/utils/resourceActionsReact';
 
 export const ResourceCardMenu: React.FunctionComponent<{
   disabled: boolean, resourceType: ResourceType, onRenameResource?: Function,
-  onCloneResource?: Function, onCopyConceptSet?: Function, onDeleteResource?: Function,
-  onEdit?: Function, onExportDataSet: Function, onReviewCohort?: Function,
+  onCloneResource?: Function, onCopyConceptSet?: Function, canDelete: boolean,
+  onDeleteResource?: Function, canEdit: boolean, onEdit?: Function,
+  onExportDataSet: Function, onReviewCohort?: Function,
```

Comment:
nit: I can't wrap my head around multiple lambdas in one line, particularly when they're mixed in with multiple assignments. I'd put one property per line. (Does the linter have an opinion on this?)

Reply:
The linter has no opinion on this aside from 'don't have lines longer than 100 characters' which is the reason I shuffled these.